api/19/src/test/java/com/toasttab/android/Api19DexTest.kt

```diff
@@ -1,12 +1,13 @@
 package com.toasttab.android
 
-import com.google.common.truth.Truth.assertThat
 import com.toasttab.android.signature.test.D8Runner
 import org.junit.Test
+import strikt.api.expectThat
+import strikt.assertions.isEmpty
```

Comment:
Because why should they use the standard method name, and why should they use an instance method?